src/Bicep.LangServer/Handlers/InsertResourceHandler.cs

```diff
@@ -50,20 +50,26 @@ public class InsertResourceHandler : IJsonRpcNotificationHandler<InsertResourceP
         private readonly ILanguageServerFacade server;
         private readonly ICompilationManager compilationManager;
         private readonly IAzResourceProvider azResourceProvider;
-        private readonly IAzResourceTypeLoader azResourceTypeLoader;
+        private readonly IAzResourceTypeLoaderFactory azResourceTypeLoaderFactory;
         private readonly TelemetryAndErrorHandlingHelper<Unit> helper;
 
-        public InsertResourceHandler(ILanguageServerFacade server, ICompilationManager compilationManager, IAzResourceProvider azResourceProvider, IAzResourceTypeLoader azResourceTypeLoader, ITelemetryProvider telemetryProvider)
+        public InsertResourceHandler(
```

Comment:
>InsertResourceHandler [](http://example.com/codeflow?start=15&length=21)

I forget if we have coverage for insert resource in the VS code tests. If not, have you tried out the feature manually?

Reply:
not sure, I will likely need some pairing to understand what to test and how

Reply:
Start with step 6 in this article: https://www.kallemarjokorpi.fi/blog/azure-bicep-insert-resource-feature-in-vs-code.html or try this article: https://luke.geek.nz/azure/azure-bicep-and-insert-resource/